common/scala/src/main/scala/whisk/core/entity/Exec.scala

```diff
@@ -177,11 +177,11 @@ protected[core] object Exec
                 val base = Map("kind" -> JsString(Exec.PYTHON), "code" -> JsString(code), "binary" -> JsBoolean(p.binary))
                 main.map(m => JsObject(base + ("main" -> JsString(m)))).getOrElse(JsObject(base))
 
-            case SequenceExec(code, comp) =>
-                JsObject("kind" -> JsString(Exec.SEQUENCE), "code" -> JsString(code), "components" -> comp.map(_.qualifiedNameWithLeadingSlash).toJson)
+            case SequenceExec(comp) =>
+                JsObject("kind" -> JsString(Exec.SEQUENCE), "components" -> comp.map(_.qualifiedNameWithLeadingSlash).toJson)
 
             case b: BlackBoxExec =>
-                val base = Map("kind" -> JsString(Exec.BLACKBOX), "image" -> JsString(b.image), "binary" -> JsBoolean(b.binary))
+                val base = Map("kind" -> JsString(Exec.BLACKBOX), "image" -> JsString(b.image.get), "binary" -> JsBoolean(b.binary))
```

Comment:
Please add a comment here and above in `BlackBoxExec` when using `image.get` stating why its safe to do so and that it'll never throw an exception.